src/schemas/string.ts

```diff
@@ -3,6 +3,7 @@ import * as Yup from "yup";
 
 export const junoAddrPattern = /^juno1[a-z0-9]{38,58}$/i;
 export const terraAddrPattern = /^terra1[a-z0-9]{38}$/i;
+export const alphanumeric = /^[0-9a-zA-Z]+$/;
```

Comment:
A thought for some future PR, maybe a better place for these regexes would be `/src/constants/regex.ts` or something in that vein, so as not to confuse regex with Yup schemas.